operations/host_spawn.go

```diff
@@ -641,3 +641,44 @@ func hostTerminate() cli.Command {
 		},
 	}
 }
+
+func hostRunCommand() cli.Command {
+	const commandFlagName = "command"
+
+	return cli.Command{
+		Name:  "run-command",
+		Usage: "run a bash shell script on the host",
+		Flags: addHostFlag(
+			cli.StringFlag{
```

Comment:
Would it also make sense to be able to read the shell script from a file?

Reply:
done.
I made a script passed in take precedence to a path, since urfave/cli has no way to make two flags exclusive 🙁 and this way made more sense than the other.

Reply:
There is a merge function called requireFileExists which may be relevant here.

Reply:
Is it necessary to handle that error separately? In other words, catching the error reading in the file should handle whether the file exists as well as any other issue such as permissions.

Additionally, if the user passes in both a script and a file (for reasons known only to them) I would want it to use the script and not error if the file doesn't exist given that the script has precedence.

Reply:
You could handle the pathFlagName/scriptFlagName XOR logic by writing a cli.BeforeFunc that does a check like `if c.String(scriptFlagName) != "" && c.String(pathFlagName) != "" { return <some_error> }`

Reply:
I think it would be clearer to return an explicit error in the case both are passed, so the user doesn't get an OK with unexpected behavior.

Reply:
done